python/py-virtualenv/Portfile

```diff
@@ -22,38 +22,31 @@ homepage            https://virtualenv.pypa.io
 master_sites        pypi:v/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  1ec5152c8d82350fd24358cfa0d112fdf70e7094 \
-                    sha256  0d62c70883c0342d59c11d0ddac0d954d0431321a41ab20851facf2b222598f3 \
-                    size    5121717
+checksums           rmd160  a3d2e5b06a9bfbc366571961966ccea70a74daca \
+                    sha256  7e4e234e1f27755685dc54063d989756790b0682d60e304db589fa1604938013 \
+                    size    7972183
 
 # keep older Python versions here, do add the EOL version to the list below
 python.versions     27 34 35 36 37 38
 
 if {${name} ne ${subport}} {
     # add EOL warnings
-    if {${python.version} in "34"} {
+    if {${python.version} in {"27" "34"}} {
         PortGroup   deprecated 1.0
         deprecated.eol_version  yes
     }
 
-    depends_lib-append  port:py${python.version}-setuptools
+     depends_lib-append     port:py${python.version}-appdirs \
+                            port:py${python.version}-distlib \
+                            port:py${python.version}-filelock \
+                            port:py${python.version}-importlib-metadata \
+                            port:py${python.version}-setuptools 
 
     depends_run-append  port:virtualenv_select
 
     select.group        virtualenv
     select.file         virtualenv${python.version}
 
-    post-destroot {
-        set docdir ${prefix}/share/doc/${subport}
-        xinstall -d ${destroot}${docdir}/docs/changelog
-        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE.txt AUTHORS.txt \
-            ${destroot}${docdir}
-        xinstall -m 0644 {*}[glob -directory ${worksrcpath}/docs *.rst] \
-                ${destroot}${docdir}/docs
-        xinstall -m 0644 {*}[glob -directory ${worksrcpath}/docs/changelog *] \
-                ${destroot}${docdir}/docs/changelog
-    }
```

Comment:
please keep the ```post-destroot``` block, just update it so that whatever files there are in the new version get installed

Reply:
I checked and there wasn't any files that needed to get installed.

Reply:
There is  a ```LICENSE``` and ```README.md``` file and also the ```docs``` directory is still present. So there isn't a real need to change this, or do you find them not useful? I would say there is no harm in installing the first two files, but perhaps the ```.rst``` documentation files and stuff in the ```changelog``` directory aren't all that useful for an end-user. What do you think?

Reply:
I tried to do `sudo port destroot py37-virtualenv`, and these files were not present. I'm doing something wrong? 

```
$> find /opt/local/var/macports/build/_Users_rubendibattista_git_macports-ports_python_py-virtualenv/py37-virtualenv/work/destroot -type f | grep README
```

No output. Same for `LICENSE`. Same for:
```
find /opt/local/var/macports/build/_Users_rubendibattista_git_macports-ports_python_py-virtualenv/py37-virtualenv/work/destroot -type d | grep docs
```

Reply:
If you don't add the ```post-destroot``` block the file(s) will indeed not be present in the ```destroot``` directory (they're copied in that block to the ```destroot``` dir). 

I would do ```sudo port extract py37-virtualenv``` to fetch/extract the archive that will be used for the installation, the you can do ```cd port `dir py37-virtualenv`/work``` and ```cd``` into the directory that is present there. You can check which files are present there and might be useful to install in the ```post-destroot``` block. Additionally, you'll find the ```setup.py``` and/or ```setup.cfg``` file that you can use double-check the dependencies. Hope that makes sense?

Reply:
Very clear. I thought that port destroot would stop after destroot (and before post-destroot).

Thanks!